lib/potassium/recipes/api.rb

```diff
@@ -31,4 +43,73 @@ def add_api
       generate "power_api:install"
     end
   end
+
+  def add_graphql
```

Comment:
Assignment Branch Condition size for add_graphql is too high. [25.1/25]<br>Method has too many lines. [62/15]